### ISSUE TITLE
False positive: lint unaware of empty function definition

### ISSUE THREAD

**Issue by TotalVerb** [tonyhffong/Lint.jl#185]
```
julia> lintstr("""
       module Test
       function test end
       export test
       end
       """)
1-element Array{Lint.LintMessage,1}:
 none:3 W361 test: exporting undefined symbol
```
